src/runtime_src/xrt/device/hal2.cpp

```diff
@@ -459,7 +459,11 @@ exec_wait(int timeout_ms) const
 {
   auto retval = m_ops->mExecWait(m_handle,timeout_ms);
   if (retval==-1)
```

Comment:
Larry.  Add curly braces around outer if.   Compiler barfs without it  for standard warnings.

Reply:
fixed.